README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with Stellarstation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [Stellarstation](stellarstation.com) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
+
+```bash
+$ pip install conan
+```
+
+For alternative methods to install Conan, see the [Conan documentation](https://docs.conan.io/en/latest/installation.html).
+
+After installing Conan, run the following commands to add the relevant Conan recipes:
+
+```bash
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan remote add inexorgame https://api.bintray.com/conan/inexorgame/inexor-conan
+$ conan remote add stellarstation https://api.bintray.com/conan/infostellarinc/stellarstation-conan
+```
+
+Proceed to install gr-stellarstation normally with CMake:
+
+```bash
+$ git clone https://github.com/infostellarinc/gr-stellarstation.git
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
+$ make
+$ sudo make install
+$ sudo ldconfig
+```
+
+## Examples
+
+See the [examples directory](examples/) for flowgraphs showing the usage of gr-stellarstation blocks.
+
+## Stellarstation API Key
+
+To use most of the blocks in this module, you will need a Stellarstation API Key, which is a .json file containing credentials to access the Stellarstation API. You can request one from <https://www.stellarstation.com/>.
```

Comment:
I feel like we need to give more details here but I don't know what our process for requesting API keys is.

Reply:
Before we open Amateur API to more people , I think we can just say "An API key can be generated in StellarStation Console."